src/screens/Exchange/Exchange.js

```diff
@@ -691,12 +730,34 @@ class ExchangeScreen extends React.Component<Props, State> {
             </CardInnerRow>
           </CardRow>
           <CardRow>
+            {!!isFiat &&
+            <CardColumn style={{ flex: 1 }}>
+              <CardText label>Fees Total</CardText>
+              <View style={{ flexDirection: 'row' }}>
+                <CardText>{feeAmount !== '' ? `${feeAmount + extraFeeAmount} ${fromAssetCode}` : 'Check'}</CardText>
```

Comment:
got fee 5.39000........1 from eur to eth on MoonPay, maybe we can format it using `formatAmountDisplay`?

Reply:
For fiat currency we can go to 2 decimals, it would be enough

Reply:
needs formatting, but agree on 2 decimals only 👍

Reply:
927faae